web_tree_dynamic_colored_field/README.rst

```diff
@@ -0,0 +1,105 @@
+Colorize field in tree views
```

Comment:
update readme according to https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst